pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -374,3 +380,47 @@ func Write(ctx context.Context, dbInfo *DBInfo, store string, deletes storage.De
 
 	return nil
 }
+
+func IsReady(ctx context.Context, db *sql.DB) (bool, error) {
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
+	defer cancel()
+
+	if err := db.PingContext(ctx); err != nil {
+		return false, err
+	}
+
+	currentVersion, err := goose.GetDBVersion(db)
+	if err != nil {
+		return false, err
+	}
+
+	if currentVersion != latestDBVersion {
+		return false, fmt.Errorf("database version is %d but expected %d", currentVersion, latestDBVersion)
+	}
+
+	return true, nil
+}
+
+func ToUserPartsFromObjectRelation(u *openfgapb.ObjectRelation) (string, string, string) {
```

Comment:
Feels like this (and the functions below) belong in the tuple utils package (`pkg/tuple`). 

If we can, let's consolidate. We now have 3 variations of tuple transformation related functions.